### PR TITLE
Initial set up for forms

### PR DESCRIPTION
**Pull request by dgalarza**

### CODE REVIEW

File: src/Input.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  font-size: ${fontSizes.size2};
+  line-height: ${lineHeights.small};
+  padding: ${spacings.smaller};
+  width: 100%;
+
+  ${placeholder({ color: colors.grey50 })};
+
+  &:focus {
+    border: ${borderWidth.small} solid ${colors.blueLight};
+    box-shadow: 0px 0px 8px rgba(68, 85, 255, 0.1);
```
Comment (mikecx) [inline src/Input.js]: Are we allowed shorthand here? 0 0 8px v.s. 0px 0px 8px?

File: src/Input.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  &:disabled {
+    background: ${colors.white10};
+  }
+`;
+
+const Input = ({ caption, label, id, ...props }) => (
+  <div>
```
Comment (mikecx) [inline src/Input.js]: Can this be a Fragment instead or are we relying on the block styles of a div?

File: src/Input.js (hunk 1 above)
Comment (dgalarza) [reply src/Input.js]: Good call. Also going to look into some linting for something like this.

Comment (dgalarza) [reply src/Input.js]: That `rgba` should probably be using one of our colors as well.

**Pull request merged**